text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -1,6 +1,6 @@
 # Support Elastic Common Schema in OpenTelemetry
 
-Collaborators: Alolita Sharma (OTEL GC, AWS), Cyrille Le Clerc (Elastic), Daniel Khan (Elastic), Christian Beedgen (Sumo), Jonah Kowall (Logz.io), Tigran N. (Splunk)
+Collaborators: Alolita Sharma (OTEL GC, Apple), Cyrille Le Clerc (Elastic), Daniel Khan (Sentry), Christian Beedgen (Sumo), Jonah Kowall (Logz.io), Tigran N. (Splunk)
```

Comment:
Thank you for keeping me on this list but as I am not participating since joining Sentry, I think it's fair to remove me.

Reply:
done thanks